acceptance/tests/facts/windows_os.rb

```diff
@@ -0,0 +1,39 @@
+test_name "Test facter os reports correct os.windows.system32" do
+    require 'timeout'
+
+    confine :to, :platform => /windows/
+
+    def with_retry(max_retries, &block)
+        retries = 0
+        begin
+            block.call
+        rescue Beaker::Host::CommandFailure => e
+            if retries <= max_retries
+                retries += 1
+                sleep 2 ** retries
+                retry
+            else
+                raise "Timeout: #{e.message}"
+            end
+        end
+    end
```

Comment:
we could find a better way of doing this. Instead of trying to execute a ssh command, maybe we can rely on ruby to check if host and port 22 are reachable. I think that would be faster than waiting for and ssh timeout. Also i would see this moved moved in a generic module, so it could be used in other tests